lib/checks/forms/fieldset.js

```diff
@@ -50,28 +50,62 @@ function checkARIAGroup(group, name) {
 	return true;
 }
 
+function checkTable (group, nodes) {
```

Comment:
@WilcoFiers take a look at my last comment on this ticket https://github.com/dequelabs/axe-core/issues/306